Include/httpClient/httpClient.h

```diff
@@ -688,6 +688,21 @@ STDAPI HCWebSocketSetProxyUri(
     _In_z_ const char* proxyUri
     ) noexcept;
 
+#if HC_PLATFORM == HC_PLATFORM_WIN32 
```

Comment:
>#if HC_PLATFORM == HC_PLATFORM_WIN32 [](start = 0, length = 36)

This is only used for wspp correct? Perhaps this should be be 

#if HC_PLATFORM == HC_PLATFORM_WIN32 && !HC_WINHTTP_WEBSOCKETS